### PR TITLE
Core alert fields fix

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Changes to Core Alert Fields
## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/CoreAlertFields/IncidentFields/incidentfield-excluded.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "sla": 0,
     "system": false,
     "threshold": 72,
-    "type": "boolean",
+    "type": "shortText",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-excluded.json:25]: pending answer by @shaniacht1

File: Packs/CoreAlertFields/IncidentFields/incidentfield-excluded.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
...
     "unmapped": false,
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-excluded.json:26]: change to true

File: Packs/CoreAlertFields/IncidentFields/incidentfield-starred.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     "sla": 0,
     "system": false,
     "threshold": 72,
-    "type": "boolean",
+    "type": "shortText",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-starred.json:25]: pending answer by @shaniacht1

File: Packs/CoreAlertFields/IncidentFields/incidentfield-starred.json [hunk 4]
```diff
@@ -22,7 +22,7 @@
...
     "unmapped": false,
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-starred.json:26]: Change to true

File: Packs/CoreAlertFields/IncidentFields/incidentfield-starred.json (hunk 3 above)
Comment (shaniacht1) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-starred.json:25]: ithink it should be left bool

**Review by David-BMS — APPROVED** [review]

**Pull request merged**